app/client/components/payment/update/currentPaymentDetails.tsx

```diff
@@ -1,13 +1,16 @@
 import React from "react";
-import { Subscription } from "../../user";
+import { Subscription } from "../../../../shared/meProductResponse";
 import { CardDisplay } from "../cardDisplay";
+import { DirectDebitDisplay } from "../directDebitDisplay";
 import { PayPalDisplay } from "../paypalDisplay";
 
 export const CurrentPaymentDetails = (subscription: Subscription) => {
   if (subscription.card) {
     return <CardDisplay {...subscription.card} />;
   } else if (subscription.payPalEmail) {
     return <PayPalDisplay payPalEmail={subscription.payPalEmail} />;
+  } else if (subscription.account) {
```

Comment:
To remove ambiguity, I think this should be bankAccountNumber, else I'd get it confused with Zuora Billing Account. If you don't have control and Subscription is a direct mapping of members-data-api JSON then I guess we can refactor it in future once Digital Pack is also fully migrated over.

Reply:
yes this is to match members data api response, will be improved when we add more DD detail to members data api response